Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -350,6 +350,17 @@ namespace AzToolsFramework
                 return findInstancesResult->get();
             }
 
+            PrefabDomValueReference GetInstancesValue(PrefabDomValue& prefabDom)
+            {
+                PrefabDomValueReference findInstancesResult = FindPrefabDomValue(prefabDom, PrefabDomUtils::InstancesName);
+                if (!findInstancesResult.has_value() || !(findInstancesResult->get().IsObject()))
+                {
+                    return AZStd::nullopt;
+                }
+
+                return findInstancesResult->get();
+            }
+
```

Comment:
I noticed there is already a const version in development branch.

```cpp
PrefabDomValueConstReference GetInstancesValue(const PrefabDomValue& prefabDom);
```

Reply:
A non-const version was needed because the calling function also would modify the object through the reference and so const version couldn't be used.